tests/integration/route_test.py

```diff
@@ -2090,3 +2092,75 @@ def test_append_route_rule(route_rule_test_env):
         desired_state[RouteRule.KEY][RouteRule.CONFIG]
         + new_desired_state[RouteRule.KEY][RouteRule.CONFIG]
     )
+
+
+@pytest.fixture
+def cleanup_veth1_kernel_mode():
+    with disable_service("NetworkManager"):
```

Comment:
What is the reason that we have to disable NetworkManager before applying the static route in kernel mode? Will `nmstatectl apply --kernel` not work as expected if not disabling NetworkManager?

Reply:
Indicate we are not using NetworkManager to apply the network config.

In theory, with NetworkManager started, this kernel mode should also works well. Even not so, it's a bug of NM, not nmstate.